api/src/main/java/uk/gov/hmcts/auth/provider/service/api/model/TokenDetails.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.auth.provider.service.api.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.hibernate.validator.constraints.NotEmpty;
+
+public class TokenDetails {
+
+    @NotEmpty
+    public final String microservice;
+
+    @NotEmpty
+    public final String oneTimePassword;
+
+    public TokenDetails(
+        @JsonProperty("microservice") String microservice,
+        @JsonProperty("oneTimePassword") String oneTimePassword
```

Comment:
should be `one_time_password` according to hmcts standards:
https://hmcts.github.io/restful-api-standards/#118